controllers/router.js

```diff
@@ -15,33 +14,38 @@ var house = {
 };
 
 // Map routes to controller functions
-module.exports = function(app) {
+module.exports = function () {
```

Comment:
@joliveros as I've seen in `webapp.js`, my only doubt is, there a reason to favor returning a function instead of the plain object? 🤔